demo/admin/src/userGroups/UserGroupContextMenuItem.tsx

```diff
@@ -1,6 +1,15 @@
 import { messages, SelectField } from "@comet/admin";
 import { Account } from "@comet/admin-icons";
-import { Button, Dialog, DialogActions, DialogContent, DialogTitle, ListItemIcon, MenuItem } from "@mui/material";
+import {
+    Button,
+    // eslint-disable-next-line no-restricted-imports
```

Comment:
why do we need this for mui?

Reply:
ok thx @manuelblum this is the problem here https://github.com/vivid-planet/comet/pull/3253